### PR TITLE
Bump develop/alt/roubaix

### PR DESCRIPTION
**Pull request by MoretS**
Bump current 0.16 decidim version to develop version from BCN repo with new decidim modules enabled

- Feature added : Max number of vote per project on decidim-budgets module
https://github.com/armandfardeau/decidim/tree/feature/maximum-number-of-vote-per-project

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -4,49 +4,70 @@ source "https://rubygems.org"
 
 ruby RUBY_VERSION
 
-gem "decidim", git: "https://github.com/OpenSourcePolitics/decidim.git", branch: "0.16-stable"
+gem "decidim", git: "https://github.com/armandfardeau/decidim", branch: "feature/maximum-number-of-vote-per-project"
```
Comment (Quentinchampenois) [inline Gemfile:7]: Should not we create an OpenSourcePolitics branch  ?

File: Gemfile [hunk 2]
```diff
@@ -4,49 +4,70 @@ source "https://rubygems.org"
...
+
+gem "sprockets", "~> 3.7"
 
 group :development, :test do
-  gem "byebug", "~> 10.0", platform: :mri
+  gem "dotenv-rails"
+  gem "byebug", "~> 11.0", platform: :mri
 
-  gem "decidim-dev", git: "https://github.com/OpenSourcePolitics/decidim.git", branch: "0.16-stable"
+  gem "decidim-dev", git: "https://github.com/armandfardeau/decidim", branch: "feature/maximum-number-of-vote-per-project"
```
Comment (Quentinchampenois) [inline Gemfile:52]: Should not we create an OpenSourcePolitics branch ?

File: config/routes.rb [hunk 3]
```diff
@@ -1,13 +1,14 @@
...
-  authenticate :user, lambda { |u| u.roles.include?("admin") } do
+Rails.application.routes.draw do
+  authenticate :admin do
     mount Sidekiq::Web => '/sidekiq'
   end
 
-  if Rails.env.development?
-    mount LetterOpenerWeb::Engine, at: "/letter_opener"
-  end
+  mount LetterOpenerWeb::Engine, at: '/letter_opener' # if Rails.env.development?
```
Comment (Quentinchampenois) [inline config/routes.rb]: Hi @MoretS, 

Is it necessary to keep the letter_opener route for production environment ?

```suggestion
  mount LetterOpenerWeb::Engine, at: '/letter_opener' if Rails.env.development?
```

File: lib/tasks/heroku.rake [hunk 4]
```diff
@@ -1,34 +1,26 @@
...
-  desc "Deploy a test version on heroku"
+  desc 'Deploy a test version on heroku'
   task setup: :environment do
-    if ENV["SECRET_KEY_BASE"].nil?
-      puts "No SECRET_KEY_BASE found !"
-      puts "export SECRET_KEY_BASE first : "
+    if ENV['SECRET_KEY_BASE'].nil?
+      puts 'No SECRET_KEY_BASE found !'
+      puts 'export SECRET_KEY_BASE first : '
       puts "export SECRET_KEY_BASE=#{`bundle exe rake secret`}"
```
Comment (Quentinchampenois) [inline lib/tasks/heroku.rake]:
```suggestion
      puts "export SECRET_KEY_BASE=#{`bundle exec rake secret`}"
```

**Review by Quentinchampenois — CHANGES_REQUESTED** [review]

File: Gemfile (hunk 1 above)
Comment (armandfardeau) [reply Gemfile:7]: Yes we may.

File: Gemfile (hunk 2 above)
Comment (armandfardeau) [reply Gemfile:52]: +1

**Review by armandfardeau — CHANGES_REQUESTED** [review]
Fix the little issues and we are good to go

**Review by armandfardeau — APPROVED** [review]

**Review by moustachu — APPROVED** [review]

**Review by Quentinchampenois — APPROVED** [review]

**Pull request closed without merging**